qemu/tests/cfg/driver_load_stress.cfg

```diff
@@ -0,0 +1,50 @@
+- driver_load_stress:
+    type = driver_in_use
+    start_vm = yes
+    kill_vm_on_error = yes
+    check_guest_bsod = yes
+    kill_vm_on_error = yes
+    login_timeout = 240
+    suppress_exception = False
+    run_bg_flag = "before_bg_test"
+    sleep_time = 300
+    driver_load_cmd = C:\devcon.exe enable @DRIVER_ID
+    driver_unload_cmd = C:\devcon.exe disable @DRIVER_ID
+    driver_id_cmd = C:\devcon.exe find * | find "VirtIO"
+    only Windows
+    variants:
+        - with_block:
+            sub_test = driver_load
+            driver_name = viostor
+            run_bg_stress = iozone_windows
```

Comment:
Hi scui,
netdev, baloon, rng, serial drivers will be tested as well.

Regards,